src/window.ui

```diff
@@ -598,17 +598,24 @@ along with Video Downloader.  If not, see <http://www.gnu.org/licenses/>.
                 <property name="can-focus">False</property>
                 <property name="border-width">18</property>
                 <property name="orientation">vertical</property>
-                <property name="spacing">6</property>
+                <property name="spacing">8</property>
                 <child>
                   <object class="GtkBox">
                     <property name="visible">True</property>
                     <property name="can-focus">False</property>
-                    <property name="spacing">12</property>
+                    <property name="halign">center</property>
+                    <property name="valign">center</property>
+                    <property name="margin-bottom">6</property>
```

Comment:
Is it intentional that the margin above and below `[Download failed]` are different? On the success page the margin between `[Download finished]` and the button is 24 px.

```
[Icon]
24 px
[Download failed]
(8+6) px
[Details]
8 px
…
```

Reply:
Nop, they aren't... I thought I had fixed, but apparently lost the change when switching hard drives.